### PR TITLE
Add Spree::Order#shipping_method

### PR DESCRIPTION
**Pull request by mbj**
* Only replaces call sides in views where `Shipment#shipping_method` was used.

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ steps:
       queue: mrh-build
     timeout_in_minutes: 10
 
-  - name: ':skull: Spree::Order'
+  - name: ':skull: Spree::Order*'
```
Comment (dkubb) [inline .buildkite/pipeline.yml]: I noticed that the command is for `Spree::Order` not `Spree::Order*`. Should the command be changed to correspond with this name?

File: backend/app/views/spree/admin/orders/_line_items.html.erb [hunk 2]
```diff
@@ -50,13 +50,13 @@
...
-      <% order.shipments.select(&:selected_shipping_rate).each do |shipment| %>
+      <% if order.shipping_method %>
         <tr class="show-method total">
           <td colspan="4">
-            <strong><%= shipment.selected_shipping_rate.name %></strong>
+            <strong><%= order.shipping_method.name %></strong>
           </td>
           <td class="total" align="center">
-            <span><%= shipment.display_cost %></span>
+            <span><%= Spree::Money.new(order.shipment_total, currency: order.currency).to_html %></span>
```
Comment (dkubb) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: At first I was going to say this could be moved into it's own commit, but I realize that this view changes the representation of the shipping method from the shipment-level to the order-level, so asking the order for the shipment total is an appropriate change to make in this commit.

File: core/spec/models/spree/order_spec.rb [hunk 3]
```diff
@@ -557,6 +557,51 @@ def assert_expected_order_state
...
+        it 'returns that shipping method' do
+          expect(apply).to be(
+            order.shipments.first.selected_shipping_rate.shipping_method
+          )
+        end
+      end
+    end
+
+    context 'without shipping rates' do
+      subject(:order) { create(:completed_order) }
```
Comment (dkubb) [inline core/spec/models/spree/order_spec.rb]: Can this use `build(:completed_order)` and offer the same amount of coverage?

File: frontend/app/views/spree/orders/_adjustments.html.erb [hunk 4]
```diff
@@ -5,11 +5,11 @@
     <% end %>
   <% end %>
 
-  <% @order.shipments.each do |shipment| %>
+  <% if @order.shipping_method %>
     <tr>
-      <td colspan="4" align='right'><h5><%= Spree.t(:shipping) %>: <%= shipment.shipping_method.name %></h5></td>
+      <td colspan="4" align='right'><h5><%= Spree.t(:shipping) %>: <%= @order.shipping_method.name %></h5></td>
       <td colspan='2'>
```
Comment (dkubb) [inline frontend/app/views/spree/orders/_adjustments.html.erb]: I realize this isn't part of the change, but we should generally use double quotes for HTML attributes.

File: core/spec/models/spree/order_spec.rb (hunk 3 above)
Comment (mbj) [inline core/spec/models/spree/order_spec.rb]: No, the factories do not work as `after` hooks require DB reads. This is fixed in the branch I extracted from.

File: backend/app/views/spree/admin/orders/_line_items.html.erb (hunk 2 above)
Comment (mbj) [inline backend/app/views/spree/admin/orders/_line_items.html.erb]: Hard, actually. Before my branch I extracted this from the `shipment_total` is not guaranteed to be correct
